pages/people.md

```diff
@@ -18,6 +18,7 @@ permalink: /people/
 | [Drew Farris](https://github.com/drewfarris)                | [Booz Allen Hamilton][bah]          | [ET][et]   |
 | [Joseph Koshakow](https://github.com/jkosh44)               |                                     | [ET][et]   |
 | [Josh Elser](https://github.com/joshelser)                  | [Hortonworks][hw]                   | [ET][et]   |
+| [Karthick Narendran](https://github.com/karthick-rn)        | [Microsoft][msft]                   | [BST][bst] |
```

Comment:
It looks like you are the first to use that Timezone.  Need to add the timezone link at the end of the file.

Reply:
Thanks for pointing this. I have updated the TZ link.